spec/support/factory_girl.rb

```diff
@@ -0,0 +1,15 @@
+require 'factory_girl'
+
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
Brought this in, this allows you to use create/build instead of namespacing them like FactoryGirl.create/FactoryGirl.build.

Also it's a good practice to lint your factories, see commented code.